pkg/remote/terraform/provider.go

```diff
@@ -128,11 +128,11 @@ func (p *TerraformProvider) configure(alias string) error {
 		"alias": alias,
 	}).Debug("New gRPC client started")
 
-	output.Printf("Terraform provider initialized (name=%s", p.Config.Name)
+	logrus.Debugf("Terraform provider initialized (name=%s", p.Config.Name)
```

Comment:
You use `logrus.WithFields` instead of adding context values directly in the string. It's a better way to make our logs more structured and readable.

Reply:
You're right, the previous code was buggy anyway.